questions/questions/intervals.py

```diff
@@ -109,3 +109,42 @@ def _generate_question_type(self):
 
     def _generate_question_params(self):
         self._question_params = {}
+
+
+class InvertedInterval(Question):
+    interval_qualities = ['Major', 'Minor', 'Augmented', 'Diminished', 'Perfect']
+    interval_numbers = [2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15]
```

Comment:
Let's use range() here so we can easily change this and not hard code the values.